ui-admin/src/util/timeUtils.tsx

```diff
@@ -6,8 +6,12 @@ export function instantToDefaultString(instant?: number) {
   return new Date(instant * 1000).toLocaleString()
 }
 /** renders a java LocalDate as a date string.  this should be sensitive to the user's locale */
-export function dateToDefaultString(date: number) {
-  return new Date(date).toLocaleDateString()
+export function dateToDefaultString(date?: number[]) {
```

Comment:
Suggest declaring this type as `number[] | undefined` instead of making the argument optional. As is, TS will allow calling the function without an argument (`dateToDefaultString()`), which I assume is not intended.

```suggestion
export function dateToDefaultString(date: number[] | undefined) {
```